hack/operatorhub/main.go

```diff
@@ -595,11 +600,17 @@ func getImageDigest(apiKey, projectId, version string) (string, error) {
 	req.Header.Set("X-API-KEY", apiKey)
 
 	q := req.URL.Query()
-	q.Add("filter", fmt.Sprintf("repositories.tags.name==%s;deleted==false", version))
+	// This filter does not work any longer.  Redhat case is open to find working filter.
+	// In the interim, we're going to get the most recent 4 published, undeleted images
+	// sorted by descending creation date, and loop through them.
+	// q.Add("filter", fmt.Sprintf("repositories.tags.name==%s;deleted==false", version))
+	q.Add("filter", "repositories.published==true;deleted==false")
+	q.Add("page_size", "4")
```

Comment:
Maybe make it a constant and add it in the error message below?
```suggestion
	q.Add("page_size", MAX_IMAGES)
```